dali/pipeline/operator/op_schema.h

```diff
@@ -291,6 +291,14 @@ graph even if its outputs are not used.)code", false);
     return *this;
   }
 
+  /**
+   * @brief Notes that operator's tensor arguments can be specified per-frame
+   */
+  DLL_PUBLIC inline OpSchema& SupportPerFrameArgs() {
+    support_per_frame_ = true;
+    return *this;
+  }
```

Comment:
I think what we might need fine grain control of which exact argument makes sense to be provided per frame.

Reply:
This should be inferred from argument descriptions.